third_party/BUILD

```diff
@@ -598,6 +598,17 @@ filegroup(
     ],
 )
 
+filegroup(
```

Comment:
nit: how about adding a BUILD file under `third_party/java/java_tools`? Then things are more similar to the eventual structure of the downloaded Java tools.

Reply:
I initially didn't add the BUILD file under `third_party/java/java_tools` to follow the convention that most of the other tools in `third_party` follow. But I think you're right, it will make the migration to remote repositories easier. Done.